youtube_dl/extractor/tube8.py

```diff
@@ -1,86 +1,260 @@
 from __future__ import unicode_literals
```

Comment:
Although it wasn't used before, please add this (which I won't call boilerplate since it ensures that Python2 still works even if someone adds a test with weird characters):
```suggestion
# coding: utf-8
from __future__ import unicode_literals
```